src/cloud-api-adaptor/pkg/agent/update.go

```diff
@@ -130,25 +129,6 @@ func UpdateConfig(cfg *Config) error {
 		agentConfig.AaKbcParams = config.AAKBCParams
 	}
 
-	if config.AuthJson != "" {
-
-		logger.Printf("Updating image_registry_auth_file in agent config file with value\n")
-
-		// Check if authJsonFilePath exists. If it doesn't exists create the file
-
-		if _, err := os.Stat(cfg.authJsonPath); err != nil && os.IsNotExist(err) {
-			// Write the authJson to the defaultAuthJsonFilePath
-			err = os.WriteFile(cfg.authJsonPath, []byte(config.AuthJson), 0644)
-			if err != nil {
-				return fmt.Errorf("failed to write auth.json file: %s", err)
-			}
-		}
-
-		// Update the file path in the agent config
-		agentConfig.ImageRegistryAuthFile = "file://" + cfg.authJsonPath
```

Comment:
Hey Magnus, whilst I appreciate that the `image_registry_auth_file` isn't in kata-containers `main` branch (yet?). I was wondering if you had an alternative approach to get to the config auth json set as the config.file_path in image-rs without this?

Reply:
Ah, if we are able to this to kata main, I'd re-add a similar approach. I just removed it because at the moment this doesn't work and it's potentially confusing.